src/pykeen/sampling/filtering.py

```diff
@@ -52,19 +52,15 @@ def forward(
 
 
 class DefaultFilterer(Filterer):
-    """
-    The default filterer.
+    """The default filterer.
 
-    .. warning ::
-        This filterer may contain a correctness error, cf. https://github.com/pykeen/pykeen/issues/272
+    .. warning:: This filterer may contain a correctness error, cf. https://github.com/pykeen/pykeen/issues/272
     """
 
     def __init__(self, triples_factory: CoreTriplesFactory):
-        """
-        Initialize the filterer.
+        """Initialize the filterer.
 
-        :param triples_factory:
-            The triples factory.
+        :param triples_factory: The triples factory.
```

Comment:
@cthoyt any reason why the one-line version is preferred? At least for me, the indention helps to quickly find the matching parameter, when looking at the documentation in the source files.

Reply:
I think it's more compact when trying to read the code itself. I guess I should just leave it the way you did it, sorry

Reply:
I put it back